app/hotels/src/filter/stars/StarsFilter.js

```diff
@@ -58,3 +62,5 @@ export default class StarsFilter extends React.Component<Props, State> {
     );
   }
 }
+
+StarsFilter.isActive = (stars: Stars) => stars.length > 0;
```

Comment:
Is it a good idea to have half of the methods in the class and half of the methods outside of the class? Is it well readable for others? Will everyone expect to have additional methods somewhere in this file?

Reply:
thx for input, that's exactly one of the reasons what I couldn't decide and created PR - it's logic related to that specific component vs. code that is split across files

Reply:
I mean the design is probably alright:

- stripe is responsible for you know striping: being scrollable, having all the filters and sort them
- the filter itself is responsible for being active or inactive (no reason to have it in the stripe)

So my point is only the code readability - I would prefer `static isActive` method instead of attaching it like this. Otherwise, I think it's alright (read I don't know how to do it better)... 🙂